src/sentry/tasks/store.py

```diff
@@ -383,7 +383,7 @@ def save_event(cache_key=None, data=None, start_time=None, event_id=None,
 
     try:
         manager = EventManager(data)
-        event = manager.save(project_id)
+        event = manager.save(project_id, assume_normalized=True)
```

Comment:
This is becoming slightly dangerous now because nothing is asserting that only normalized data goes in the queue and all tests will normalize with this. If there is a bug, we can only find it in production. 

This is not a problem of your change, though it is uncovered now. I just have no idea how to fix this.

Reply:
This is already the case.